public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx

```diff
@@ -229,7 +246,15 @@ export function RichHistoryQueriesTab(props: RichHistoryQueriesTabProps) {
                 <div className={styles.heading}>
                   {heading}{' '}
                   <span className={styles.queries}>
-                    {partialResults ? 'Displaying ' : ''}
+                    {partialResults ? (
+                      <Trans
+                        i18nKey="explore.rich-history-queries-tab.displaying-queries"
+                        defaults="Displaying {{ count }} queries"
+                        values={{ count: mappedQueriesToHeadings[heading].length }}
+                      />
+                    ) : (
+                      ''
+                    )}
                     {mappedQueriesToHeadings[heading].length} queries
```

Comment:
We need to show a different message based on `partialResults` and it needs to be translated as well. The non-partial message is also out of the `partial ? (...) : (...)`, should be pulled in, otherwise both are shown when `partialResults = true`:

<img width="327" alt="Screenshot 2023-10-19 at 11 25 15" src="https://github.com/grafana/grafana/assets/745532/47398d5e-449f-4756-a962-3e32c2fb6223">


```suggestion
                    {partialResults ? (
                      <Trans
                        i18nKey="explore.rich-history-queries-tab.displaying-partial-queries"
                        defaults="Displaying {{ count }} queries"
                        values={{ count: mappedQueriesToHeadings[heading].length }}
                      />
                    ) : (
                      <Trans
                        i18nKey="explore.rich-history-queries-tab.displaying-queries"
                        defaults="{{ count }} queries"
                        values={{ count: mappedQueriesToHeadings[heading].length }}
                      />
                    )}
```